### DISCUSSION TITLE
forum-session-AMA

### DISCUSSION THREAD

**Discussion by giscus** [arXiv/accessibility2024#128] [General]
# forum-session-AMA

One of the biggest accessibility barriers is social: the discomfort that gets in the way of connection. We are often not sure what we can or should ask. Three brilliant researchers are here to cut through the fog. There are no silly questions! Come engage and learn.

https://accessibility2024.arxiv.org/forum-session-AMA

**Comment by SBBCornell** [comment]
Welcome everyone! Please share thoughts, questions, and resources of interest to the community.

**Comment by dashvonriprock** [comment]
Dyslexia effects over a 1.3 billion, I think I am the only one who has created a search system which eliminate dyslexia as an issue. Is there anyone else doing work on this?

**Reply by car222222** [reply]
Interesting: how does it achieve this?

**Comment by LaBatata** [comment]
Hello! I'm a visually impaired person currently pursuing my Masters in Mechanical Sciences, and I am excited to join this conversation tomorrow! Specially, I would love to know more about the professors' experiences in making STEM education accessible to visually impaired folks.

**Reply by car222222** [reply]
Which professors do you wish to ask?

**Reply by LaBatata** [reply]
There are a few quesitons I would like to ask professors Potluri and Godfrey! What about you?

**Comment by car222222** [comment]
Will this whole forum be inaccessible if one is in the wrong time zones?  
For example, this one starts in the middle of the night for many would be attendees!  
Not very friendly.

**Reply by JenM00re** [reply]
I believe videos will be provided. Unfortunately, the hours were bound to be not great for some people.

**Reply by sdorphan** [reply]
Recordings will be made available after the event. We did stagger times throughout the course of the event, but it's always tricky to find times that work for staff available to work the event as well as all potential attendees.

**Comment by SBBCornell** [comment]
Hi friends! Many great questions were asked at the session that we did not have time to cover. Our panelists are reviewing them, though, and we will post their answers here as they come in. See below.

**Comment by SBBCornell** [comment]
Question: How do you interact with plots?  Are there ways to make plots more accessible?

Answer from Jonathan: Yes, there are tools to help with plots. In my discipline, a plot gets made for two purposes. Most are made for the analyst as a tool to understand data and a minority of those are then used for communicating with an audience. I find ways to achieve the same ends for the first problem, often using ideas that were prominent well before graphs were so easily created. Once I need a graph for an audience, I must be able to create that graph to the standard my audience expects. This is what my R package (BrailleR) tries to provide.

**Reply by choav** [reply]
Hi Jonathan,

I am a visually impaired data scientist and hence finding ways to quickly analyze data without plots are of great interest to me. Could you go into a bit more detail on this?

I assume you use various statistical quantities as a replacement for plots to explore data? Are there tools which you find particularly useful? I would guess there are situations in which some details are lost with a non-visual description, e.g. complex behaviors of time series?

**Reply by SBBCornell** [reply]
Answer to the original question from Venkatesh:
The most common way I interact with plots is through good alternative text. Though this is not the most efficient, alternative text is relatively easy to include, is standardized across platforms, and is relatively easy to author. Having access to the data that forms the plot along with the code that generates it may also be helpful.

**Comment by SBBCornell** [comment]
Question: Do any of you who use screen readers or other visual assistive tech have a preference between accessible PDFs or a dynamic wikis (eg Confluence) for documentation?

Answer from Jonathan:  I have a complete aversion to anything pdf related. I know there are plenty of pdf documents I receive which are easily managed by a screen reader, but the experience is a distant third to the experience I have with HTML (#1 by far) and original source files such as .tex or .docx (pretty much equivalent to me).  I avoid the systems that rely on pdf including publishers or document archive services. I remind conference organisers that they need to give me an accessible version of the programme. I make regular use of the conference help desk staff when their conference material is in an inaccessible format. In short, I try to make use of pdf the problem of the provider instead of my problem as a consumer.

**Reply by SBBCornell** [reply]
Answer from Venkatesh: PDF readability is some times inconsistent with different tools. I personally use the following strategies (in no particular order of preference):

1. try a different PDF reader. I switch between acrobat reader, google chrome, and Edge.
2. try to convert the paper to HTML. there are websites like papertohtml.org that help with this. Not perfect, but some times manageable.
3. use web previews when possible. Arxiv has HTML papers now for more recent drafts. replacing the x in the URL with the number 5 opens the same paper in HTML.

**Comment by SBBCornell** [comment]
Question: I would love to know more about the professors' experiences in making STEM education accessible to visually impaired.

Venkatesh: this is a broad-ranging question with so many moving parts. Personally, I have touched many aspects to making stem education accessible – ranging from research projects that are out there, to advocacy, to taking a start-up route on this theme. So it really depends on the position that I (or you) are in, how it overlaps with education, and what we can do to improve accessibility.

**Comment by SBBCornell** [comment]
Question: Thank you for this session! I am currently working towards a career in Accessibility in STEM and User Interfaces. I would like to know what are the professors' suggestions for me. What should I learn? Where should I start?

Venkatesh: Broadly, I would encourage you to familiarize yourself with web accessibility best practices, and incorporate accessibility into your user experience designs. DQ university had some courses on accessibility – that may be another good place to start.

**Comment by SBBCornell** [comment]
Question: For me Free Open Source Software (FOSS) based systems and applications were hard to set up and get used to. Does someone of the panel have any experience with for example installing and/or using GNU/Linux, and if so, was it more or less accessible than other systems?

Venkatesh: I use GNU-Linux systems exclusively for dev work. I frequently set these up on a remote machine and access them using SSH from my primary machine, which is a windows computer running JAWS or NVDA. I additionally connect to the linux machine using VS Code server. Setup becomes easy this way as you can entirely use the command line, and you do not have to deal with potentially inaccessible graphical user interfaces. Hope this helps!

Jonathan: There are plenty of blind people using Linux, but they have more time to fight with software than I am prepared to expend. I find the same comments relevant for blind people using a Mac; they just seem to take longer than I do on Windows to complete the same task. I can, and do, use free tools all the time though in Windows. I use R daily, Python sporadically, the command line extensively, etc. I have not initiated an MS Word document or Power Point since 2015, nor any Open Office alternatives. I do use Excel though, primarily because I haven’t found an alternative that is as user-friendly and well supported by screen readers.

**Reply by SBBCornell** [reply]
Jonathan: I use JAWS 99% of the time and NVDA 1% of the time. I’ve tested the read aloud features in many tools, including ChatGPT this week. The proper screen reader software gives me increased efficiency over the read aloud tools; I did like the ChatGPT voice though and it did a lovely job of reading a bedtime story suitable for a five year old. It isn’t flexible enough to warrant its use for processing scientific material. Screen reader choice is a key decision and becoming proficient in at least one seems to be quite hit and miss. Many blind people I meet say they use JAWS or say they use NVDA, but they are not really proficient because they have only used the software in a default manner. A blind person wanting to be successful in STEM must really get on top of the tools that a screen reader offers (speed increase, tone alterations, spelling of words, verbosity settings, processing of math etc.) Each screen reader has its strengths though, so the choice of which general software a blind person prefers may be affected by the choice of screen reader they made.

**Comment by SBBCornell** [comment]
Question: Hello - and thank you so much for presenting! I am a recovering researcher... now my job is building and nurturing my University's student, faculty, and staff portals. We strive to adhere to WCAG 2.2 AA guidelines and have a user-centered design process - and we are always learning how to better present resources to all of our audiences. In that spirit, I am wondering what aspects of your student and faculty portals did or do you appreciate most - and least?

Venkatesh: I think finding a portal to be accessible to begin with is a great start, so thank you for doing this work! While I can not think of specific examples, some things that I generally liked include the use of proper heading structures to find the functionality that I would frequently want to use, and minimal changes as much as possible, to the layout of the page.
